dark/features.py

```diff
@@ -1,5 +1,20 @@
 import numpy as np
-import matplotlib.pyplot as plt
+
+try:
+    from matplotlib import pyplot as plt
+except ImportError:
+    import platform
+    if platform.python_implementation() == 'PyPy':
+        # PyPy doesn't have a version of matplotlib. Make fake classes and
+        # a Line2D function and that raise if used. This allows us to use
+        # other 'dark' code that happens to import dark.mutations but not
+        # use the functions that rely on matplotlib.
+        class plt(object):
+            def __getattr__(self, _):
+                raise NotImplementedError(
+                    'matplotlib is not supported under pypy')
+    else:
+        raise
```

Comment:
I'm not sure what you mean.  It used to just raise ImportError and quit. Now it still does that, except if you're running under PyPy.  I could have made it do the PyPy behaviour in case of any matplotlib import, but I didn't want to hide import errors that I don't know about (or hide it if the person using our code simply forgot to pip install matplotlib).